doc/decisions/0011-protocol-version-compatibility.md

```diff
@@ -0,0 +1,56 @@
+# 0011. Protocol compatibility between App versions
+
+| Date | Tags |
+|---|---|
+| Fri Oct 05  | protocol, messenger |
+
+## Status
+
+draft
+
+## Context
+
+This document enumerates the rules we have to adhere to when doing any protocol modifications and summarizes best practises to do so.
+In the light of recent issues (code landed in develop branch, producing "invisible" messages for previous app versions), it's best
+to capture it in the ADR for future reference.
+
+## Decision
```

Comment:
Not sure how, but perhaps we can format this a better nicer so it is easier to read/clearer? Feels a bit wordy right now